backend/src/main/java/com/v2gdemo/backend/service/BillingService.java

```diff
@@ -0,0 +1,38 @@
+package com.v2gdemo.backend.service;
+
+import com.v2gdemo.backend.entity.Object;
+import com.v2gdemo.backend.entity.ObjectRepository;
+import com.v2gdemo.backend.entity.Wallet;
+import com.v2gdemo.backend.entity.WalletRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BillingService {
+  private WalletRepository walletRepository;
+  private ObjectRepository objectRepository;
+  public BillingService(WalletRepository walletRepository, ObjectRepository objectRepository){
+    this.objectRepository = objectRepository;
+    this.walletRepository = walletRepository;
+  }
+  public synchronized void makeTransaction(Object car, String address, long value) throws Exception{
```

Comment:
makeTransaction(String fromWallet, String toWallet, long value)

Reply:
also API method in controller needed for makeTransaction

Reply:
K, will be done!